scripts/utilities/cellspace.py

```diff
@@ -710,10 +710,8 @@ def random_mods(self, file_prefix, mod_type='gaussian', num_mods=1, temp_mod=Fal
                 depth, _, length = self.cells.shape
                 max_length_scaling = length // image_length
                 max_depth_scaling = depth // image_depth
-                #length_scaling = random.randint(1, max_length_scaling)
-                #depth_scaling = random.randint(1, max_depth_scaling)
-                length_scaling = random.randint(max_length_scaling - 1, max_length_scaling)
-                depth_scaling = random.randint(max_depth_scaling - 1, max_depth_scaling)
+                length_scaling = random.randint(1, max_length_scaling)
```

Comment:
Add comments to explain what they space invader (and other possible outputs) is?